src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { TodoWithUser } from '../../types/types';
+
+type Props = {
+  todo: TodoWithUser;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
Not fixed
![2023-09-05_12h03_51](https://github.com/mate-academy/react_add-todo-form/assets/61904995/e3cb3fd8-a863-4d5d-9ccb-447deefc0ff9)